src/backends/pytorch/libtorch_backend.cc

```diff
@@ -605,6 +632,20 @@ LibTorchBackend::Context::Run(
 
   // Make sure each output is of the expected size and copy it into
   // the payload responses.
+
+  //  When no Output in Model Config:
```

Comment:
The outputs that we want to read and return in the response should be determined by what is in the request, not what is in the model config. I don't think this code should even look at the model config, should it? The model config should be used to make sure that only allowed outputs are requested.

Reply:
Based on the behavior of the other backends I was under the assumption that all outputs in the model config are passed to the server to finalize the response. Only while finalizing the response does the server pick only the outputs that were requested for in the request header.

Yes, it would make sense to return only those outputs to the server that the user requested. The changes would have to be made to all backends for consistency.